src/components/Overlay/Overlay.tsx

```diff
@@ -136,7 +135,7 @@ function Overlay(): JSX.Element {
           <OverlayGrid>
```

Comment:
It feels like duplication from lines 112 to 129. This is also the reason why it is not passing sonarqube. Is there a specific reason for this dupliaction? For is there any way to remove the duplication without over-commitment?

Reply:
fixed

Reply:
perfect. testing according to your list